### PR TITLE
FISH-7645 Documentation for Docker and WSL support in IDEs

### PR DESCRIPTION
**Pull request by jGauravGupta**

### CODE REVIEW

File: docs/modules/ROOT/pages/Technical Documentation/Ecosystem/IDE Integration/Eclipse Plugin/Payara Server.adoc [hunk 1]
```diff
@@ -68,6 +68,50 @@ WARNING: Changing the runtime may result in unusable state of the server because
...
+
+* Register a new Payara Server instance by performing the following:
+    ** Open the *Servers* view (if not visible, it should be under menu *Window -> Show View -> Servers*).
+    ** Select *Payara -> Payara* and set the *Host name* and some comprehensible *Server name*.
+    ** Then click *Next* to another window with server runtime configuration and browse & locate the Payara Server installation directory with the same version as the one running in the Docker container.
+    ** Then click *Next* again to window with the configuration of the domain. Enter the domain name, host, and necessary credentials.
+    ** For Docker integration, choose the "Docker Volume" option and provide the HOST_PATH (e.g., D:/test) and CONTAINER_PATH (e.g., /mnt/test) mapping details. This mapping should align with the exploded application path or parent path of exploded application.
+    ** In the last step you can select resources which will be deployed on the server or you may add it later.
+* Run the Payara Server Docker container using the following command:
+`docker run -p 8080:8080 -p 4848:4848 -p 9009:9009 -v HOST_PATH:CONTAINER_PATH  payara/server-full:latest`
```
Comment (JamesHillyard) [inline docs/modules/ROOT/pages/Technical Documentation/Ecosystem/IDE Integration/Eclipse Plugin/Payara Server.adoc:83]: The latest image would be Payara 6. This should be 5.2022.5 or use `{page-version}`

File: docs/modules/ROOT/pages/Technical Documentation/Ecosystem/IDE Integration/NetBeans Plugin/Payara Server.adoc [hunk 2]
```diff
@@ -55,6 +55,22 @@ Select the Payara Server you have configured. You can also select a context path
...
+
+* Open Apache NetBeans and navigate to the Services tab.
+* Register a new Payara Server instance by performing the following:
+    ** Click on the "Add Server" button.
+    ** Select the "Remote Domain" option.
+    ** Browse and locate the Payara Server installation directory with the same version as the one running in the Docker container.
+    ** Enter the domain name, host, and necessary credentials.
+    ** For Docker integration, choose the "Docker Volume" option and provide the HOST_PATH (e.g., D:/test) and CONTAINER_PATH (e.g., /mnt/test) mapping details. This mapping should align with the exploded application path or parent path of exploded application.
+* Run the Payara Server Docker container using the following command:
+`docker run -p 8080:8080 -p 4848:4848 -p 9009:9009 -v HOST_PATH:CONTAINER_PATH  payara/server-full:latest`
```
Comment (JamesHillyard) [inline docs/modules/ROOT/pages/Technical Documentation/Ecosystem/IDE Integration/NetBeans Plugin/Payara Server.adoc:70]: The latest image would be Payara 6. This should be 5.2022.5 or use {page-version}

File: docs/modules/ROOT/pages/Technical Documentation/Ecosystem/IDE Integration/VSCode Extension/Payara Server.adoc [hunk 3]
```diff
@@ -51,6 +51,49 @@ Deployed applications can be opened in the browser, undeployed or disabled by ri
...
+The Payara Server tools in the VS Code extension includes Docker integration, making it seamless to deploy exploded Jakarta applications to a Payara Server instance running within a local Docker container. To take advantage of this feature, follow the steps below:
+
+* Register a new Payara Server instance by performing the following:
+    ** Open the command pallet using *Ctrl + Shift + P*, type *Payara* and select the *Add Payara Server* option.
+    ** Select Remote Domain.
+    ** For Docker integration, choose the "Docker" type and provide the HOST_PATH (e.g., D:/test) and CONTAINER_PATH (e.g., /mnt/test) mapping details. This mapping should align with the exploded application path or parent path of exploded application.
+    ** Choose a name for the server.
+    ** Enter the domain name, host name, and necessary credentials.
+* Run the Payara Server Docker container using the following command:
+`docker run -p 8080:8080 -p 4848:4848 -p 9009:9009 -v HOST_PATH:CONTAINER_PATH  payara/server-full:latest`
```
Comment (JamesHillyard) [inline docs/modules/ROOT/pages/Technical Documentation/Ecosystem/IDE Integration/VSCode Extension/Payara Server.adoc:66]: The latest image would be Payara 6. This should be 5.2022.5 or use {page-version}

**Review by JamesHillyard — CHANGES_REQUESTED** [review]

**Review by fturizo — APPROVED** [review]
I made some layout changes to keep the style consistent. Good work!

**Pull request merged**